operations/app/src/modules/key_vault/main.tf

```diff
@@ -1,234 +1,234 @@
+  location                        = var.location
+  resource_group_name             = var.resource_group
+  sku_name                        = "premium"
+  tenant_id                       = data.azurerm_client_config.current.tenant_id
+  enabled_for_deployment          = true
+  enabled_for_disk_encryption     = true
+  enabled_for_template_deployment = true
+  purge_protection_enabled        = true
+
+  network_acls {
+    bypass                     = "AzureServices"
+    default_action             = "Deny"
+    virtual_network_subnet_ids = []
+    // We're using a private endpoint, so none need to be associated
+  }
+
+  lifecycle {
+    prevent_destroy = true
+  }
+
+  tags = {
+    "environment" = var.environment
+  }
 }
 
 resource "azurerm_key_vault_access_policy" "dev_access_policy" {
-    key_vault_id = azurerm_key_vault.application.id
-    tenant_id    = data.azurerm_client_config.current.tenant_id
-    object_id    = var.aad_object_keyvault_admin
-
-    key_permissions = [
-        "Get",
-        "List",
-        "Update",
-        "Create",
-        "Import",
-        "Delete",
-        "Recover",
-        "Backup",
-        "Restore"
-    ]
-
-    secret_permissions = [
-        "Get",
-        "List",
-        "Set",
-        "Delete",
-        "Recover",
-        "Backup",
-        "Restore"
-    ]
-
-    certificate_permissions = [
-        "Get",
-        "List",
-        "Update",
-        "Create",
-        "Import",
-        "Delete",
-        "Recover",
-        "Backup",
-        "Restore",
-        "ManageContacts",
-        "ManageIssuers",
-        "GetIssuers",
-        "ListIssuers",
-        "SetIssuers",
-        "DeleteIssuers"
-    ]
+  key_vault_id = azurerm_key_vault.application.id
+  tenant_id    = data.azurerm_client_config.current.tenant_id
+  object_id    = var.aad_object_keyvault_admin
+
+  key_permissions = [
+    "Get",
+    "List",
+    "Update",
+    "Create",
+    "Import",
+    "Delete",
+    "Recover",
+    "Backup",
+    "Restore"
+  ]
+
+  secret_permissions = [
+    "Get",
+    "List",
+    "Set",
+    "Delete",
+    "Recover",
+    "Backup",
+    "Restore"
+  ]
+
+  certificate_permissions = [
+    "Get",
+    "List",
+    "Update",
+    "Create",
+    "Import",
+    "Delete",
+    "Recover",
+    "Backup",
+    "Restore",
+    "ManageContacts",
+    "ManageIssuers",
+    "GetIssuers",
+    "ListIssuers",
+    "SetIssuers",
+    "DeleteIssuers"
+  ]
 }
 
 resource "azurerm_key_vault_access_policy" "frontdoor_access_policy" {
-    key_vault_id = azurerm_key_vault.application.id
-    tenant_id    = data.azurerm_client_config.current.tenant_id
-    object_id    = local.frontdoor_object_id
-
-    secret_permissions      = [
-        "Get"]
-    certificate_permissions = [
-        "Get"]
+  key_vault_id = azurerm_key_vault.application.id
+  tenant_id    = data.azurerm_client_config.current.tenant_id
+  object_id    = local.frontdoor_object_id
+
+  secret_permissions = [
+  "Get"]
```

Comment:
What does the formatting do if you make this:
```hcl
secret_permissions = [
  "Get"
]
```
??

That might be a better way to do lists.

Reply:
Agreed, I fixed up all of these instances throughout the formatted code and ensured the last item has a trailing `,`.